ElectronicsStore/MobilePhone.cs

```diff
@@ -0,0 +1,33 @@
+﻿using ElectronicsStore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AqaLabHomeworkFilesStorage.Task7
+{
+    public class MobilePhone : ElectricalDevice, ITakePhoto
+    {
+        public double NumberOfPixelsInCamera { get; set; }
```

Comment:
Also `Polaroid` class has this property. Need to fix) I can suggest creating another one abstract class `PhotoCamera`